src/wireviz/Harness.py

```diff
@@ -344,6 +345,44 @@ def create_graph(self) -> Graph:
             dot.node(cable.name, label=f'<\n{html}\n>', shape='box',
                      style=style, fillcolor=translate_color(bgcolor, "HEX"))
 
+        def typecheck(name: str, var, type) -> None:
+            if not isinstance(var, type):
+                raise Exception(f'Unexpected value type of {name}: {var}')
```

Comment:
```suggestion
        def typecheck(name: str, var, typ) -> None:
            if not isinstance(var, typ):
                raise Exception(f'Unexpected value type of {name}: Expected {typ}, got {type(var)}\n{var}')
```

Renaming `type` to `typ` to avoid using the name of the built-in `type()` function.

Reply:
```suggestion
        def typecheck(name: str, var, type_) -> None:
            if not isinstance(var, type_):
                raise Exception(f'Unexpected value type of {name}: Expected {type_}, got {type(var)}\n{var}')
```

Apparently, [PEP 8](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles) recommends single trailing underscore for these situations.